MVP Module/Tests/Feed UI/FeedUIIntegrationTests.swift

```diff
@@ -280,6 +280,29 @@ final class FeedUIIntegrationTests: XCTestCase {
 		wait(for: [exp], timeout: 1.0)
 	}
 
+	func test_loadFeedCompletion_rendersErrorMessageOnErrorUntilNextReload() {
+		let (sut, loader) = makeSUT()
+
+		sut.loadViewIfNeeded()
+		loader.completeFeedLoadingWithError()
```

Comment:
It should be tested that errorView is not visible when the view is loaded
```suggestion
		sut.loadViewIfNeeded()
		XCTAssertEqual(sut.isErrorVisible, false)
		loader.completeFeedLoadingWithError()
```